### PR TITLE
CIV-10069 Refactor Judicial options

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Screenshot attached for the Evidence of manual testing
- [x] Functional, Smoke and API tests have been run locally
- [x] Full gradle build and run tests with coverage have been completed (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-10069

### Change description ###

Update judicial make decision options

### Testing branch and instruction ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Review by pliao-hmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/enums/dq/GAJudgeDecisionOption.java [hunk 1]
```diff
@@ -6,7 +6,8 @@
 @Getter
 @RequiredArgsConstructor
 public enum GAJudgeDecisionOption {
-    MAKE_AN_ORDER("Make an order"),
+    MAKE_AN_ORDER("Make an order without a hearing"),
```
Comment (vasudevganesanhmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/enums/dq/GAJudgeDecisionOption.java:9]: there needs to be test changes done on ga ccd . . then raise a test PR in ga ccd pointing to this image . .thanks

Comment (karthick-mohanasundaram-HMCTS) [reply src/main/java/uk/gov/hmcts/reform/civil/enums/dq/GAJudgeDecisionOption.java:9]: Done

**Review by deepthidoppalapudihmcts — APPROVED** [review]

**Pull request merged**